### PR TITLE
Standardize gogoproto options

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: config/config.proto [hunk 1]
```diff
@@ -25,6 +25,7 @@ import "gogoproto/gogo.proto";
 option (gogoproto.sizer_all) = true;
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
```
Comment (petermattis) [inline config/config.proto]: What does this option do?

Comment (tamird) [inline config/config.proto]: Removes the unrecognized value passthrough

Comment (petermattis) [inline config/config.proto]: Can you add some justification to the commit message about why we want to remove unrecognized value support?

**Comment by petermattis** [comment]
LGTM

File: config/config.proto (hunk 1 above)
Comment (tamird) [inline config/config.proto]: done

**Pull request merged**